client/src/components/DisplayTable.js

```diff
@@ -79,13 +96,27 @@ export class DisplayTable extends React.Component {
                       <th 
                         key={i} 
                         className={classNames("center-text", "display-table__sortable")}
-                        onClick={ () => _.includes(sort_keys,tick) && this.header_click(tick) }
                       >
                         {table_data_headers[i]}
-                        <SortDirections 
-                          asc={!descending && sort_by === tick} 
-                          desc={descending && sort_by === tick}
-                        />
+                        <div
+                          onClick={ () => _.includes(sort_keys,tick) && this.header_click(tick) }
+                        >
+                          <SortDirections 
+                            asc={!descending && sort_by === tick} 
+                            desc={descending && sort_by === tick}
+                          />
+                        </div>
+                        { tick === "indicator" &&
```

Comment:
Hardcoding this in DisplayTable.js means we can't use DisplayTables for other data! Should probably pass in a list of filterable keys like sort_key